test_bounce.py

```
@@ -20,3 +21,24 @@
     response = client.get('/', headers={'Host': 'invalid_domain'})
     assert response.status_code == 301
     assert response.headers['Location'] == 'http://harperrules.com/'
+
+
+def test_missing_host_header(client):
+    response = client.get('/')
+    assert response.status_code == 301
+    assert response.headers['Location'] == 'http://harperrules.com/'
+
+
+def test_prometheus_counter_increment(client):
```

Ruff

Missing return type annotation for public function `test_prometheus_counter_increment`

Add return type annotation: `None`

(ANN201)